ppr-ui/src/components/mhrRegistration/YourHome/ManufacturedYearSelect.vue

```diff
@@ -75,13 +75,11 @@ export default defineComponent({
 
     const localState = reactive({
       currentYear: new Date().getFullYear(),
-      yearOfManufacture: getMhrRegistrationYearOfManufacture.value?.toString()
+      yearOfManufacture: getMhrRegistrationYearOfManufacture?.value
     })
 
-    watch(() => localState.yearOfManufacture, (val: string) => {
-      if (parseInt(val)) {
-        setMhrHomeBaseInformation({ key: 'year', value: parseInt(val) })
-      }
+    watch(() => localState.yearOfManufacture, (val: number) => {
```

Comment:
Was the Year field changed to numbers only?

Reply:
For the select yes: 

![image](https://github.com/bcgov/ppr/assets/77707952/c15d60a8-3221-4009-9c24-831288b2c1a1)

Reply:
I didn't realize this was a pre-defined select, thanks!